app/templates/components/file_upload.html

```diff
@@ -0,0 +1,74 @@
+{% macro js(file_field_id, file_url_id, preview_id) %}
+
+<script>
+
+if (!window.getSignedRequest) {
```

Comment:
Should we make a nice .js file for this so it can be used in other projects without being embedded in the HTML? Or were you thinking this would just be an example thats reproduced with jquery or react?